onward/app/weather/models/CityResponse.scala

```diff
@@ -5,17 +5,17 @@ import common.editions.{Au, Us, Uk}
 import weather.models.accuweather.LocationResponse
 import play.api.libs.json.Json
 
-import scalaz.std.AllInstances._
-import scalaz.syntax.foldable._
-
 object CityResponse {
   implicit val jsonWrites = Json.writes[CityResponse]
 
   def fromLocationResponses(locations: List[LocationResponse]) = {
     def cityAndCountry(location: LocationResponse) =
       (location.LocalizedName, location.Country.LocalizedName)
 
-    val citiesWithSameNameByCountry = locations.map({ location => Map(cityAndCountry(location) -> 1) }).suml
```

Comment:
This is one of those things that is really useful from the libraries; the `Monoid` instance for `Map`.
Nice workaround, I thought it might not be safe to remove until I seen the sneaky `getOrElse` in the return of the fold! 😄